decidim-comments/app/models/decidim/comments/comment.rb

```diff
@@ -21,9 +21,12 @@ class Comment < ApplicationRecord
       validate :commentable_can_have_replies
       validates :depth, numericality: { greater_than_or_equal_to: 0 }
       validates :alignment, inclusion: { in: [0, 1, -1] }
+      validate :same_organization
 
       before_save :compute_depth
 
+      delegate :organization, to: :commentable
```

Comment:
I added this because I added the validation that the author and commentable organization should be the same. The thing is that when replying, commentable is another comment, and comment didn't have an organization. This way when replying, the organization bubbles up to the main commentable.

If you prefer it I can change it to `has_one :organization, through: :commentable`